packages/definitions-parser/test/definition-parser.test.ts

```diff
@@ -24,7 +24,7 @@ describe(getTypingInfo, () => {
       "index.d.ts",
       `// Type definitions for @ckeditor/ckeditor-engine 25.0
 // Project: https://github.com/ckeditor/ckeditor5/tree/master/packages/ckeditor5-engine
-// Definitions by: Federico <https://github.com/fedemp>
+// Definitions by: My Self <https://github.com/ñ>
```

Comment:
I'm removing mentions of **myself** cause I found it too egocentric.
`ñ` is not a valid github username, so there's no promotion either.